validator/lib/validatePnts.js

```diff
@@ -196,6 +198,42 @@ function validatePnts(content) {
         }
     }
 
+    if (defined(featureTableJson.RGBA)) {
+        featureTable.featuresLength = pointsLength * 4;
+        componentDatatype = ComponentDatatype.fromName(defaultValue(featureTableJson.RGBA.componentType, 'UNSIGNED_SHORT', 4));
```

Comment:
`UNSIGNED_SHORT` should be `UNISNGED_BYTE`.

Reply:
We may have talked about this offline before.... but here are my latest thoughts: using a higher precision is unsafe because it distorts the values that are retrieved from the feature table binary and may result in out-of-bounds issues. Ultimately these range checks for colors are not super useful for binary properties, but are helpful for array-based properties.